drivers/tty/serdev/pibridge_trace.h

```diff
@@ -103,16 +155,37 @@ DECLARE_EVENT_CLASS(pibridge_send_crc_class,
 	)
 );
 
+/*
+ * pibridge_send_io_crc
+ *
+ * Info: the CRC of an IO packet to be sent.
+ * Time: before the CRC of an IO packet is sent on the wire.
+ */
 DEFINE_EVENT(pibridge_send_crc_class, pibridge_send_io_crc,
 	TP_PROTO(u8 crc),
 	TP_ARGS(crc)
 );
 
+/*
+ * pibridge_send_gate_crc
+ *
+ * Info: the CRC to send for the gateway packet.
+ * Time: before the CRC of a gateway packet is sent on the wire.
+ */
 DEFINE_EVENT(pibridge_send_crc_class, pibridge_send_gate_crc,
 	TP_PROTO(u8 crc),
 	TP_ARGS(crc)
 );
 
+/*
+ * pibridge_receive_crc_class
+ *
+ * Print the CRC that is part of a received packet as well as the CRC that has
+ * been calculated for this packet.
+ *
```

Comment:
It would be good to include the word "expected" here (e.g. "as well as the expected CRC") to explain the variable naming below.